src/biogeophys/WaterStateType.F90

```diff
@@ -268,6 +283,15 @@ subroutine InitHistory(this, bounds, use_aquifer_layer)
             ptr_col=this%wa_col, l2g_scale_type='veg')
     end if
 
+    ! Add excess ice fields to history
+
+    if (use_excess_ice) then
+      data2dptr => this%excess_ice_col(begc:endc,1:nlevsoi)
+      call hist_addfld2d (fname='EXCESS_ICE',  units='kg/m2', type2d='levsoi', &
```

Comment:
Because this is 3D, and not changing rapidly it should normally be on the annual file.

Reply:
We probably need to add a user-mod for CMIP output for excess ice that adds this to the annual file. I think we can do that sort of thing.